pkg/debug/Dockerfile

```diff
@@ -9,14 +9,14 @@
 # to be a perfect place to put any other kind of debug info
 # into the package: see abuild/etc/abuild.conf.
 FROM lfedge/eve-recovertpm:0da17f65aba4fb09c83944cf5847dd7b523118b4 as recovertpm
-FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 as build
+FROM lfedge/eve-alpine:43ba9262f1b3c71b7a501b549d0cfb811a2d81ee as build
```

Comment:
Please keep it consistent and also bump it everywhere else.

Reply:
You can use something like
```
find ./ -name Dockerfile -exec perl -pni.bak -e 's/lfedge\/eve-alpine(\S+)/lfedge\/eve-alpine:43ba9262f1b3c71b7a501b549d0cfb811a2d81ee/'
 "{}" \;
```
to make your life a bit easier

Reply:
do we do this across the EVE now, i thought we still selectively bump up? will search for those. thanks.